apps/web/components/booking/BookingDescription.tsx

```diff
@@ -0,0 +1,113 @@
+import { SchedulingType } from "@prisma/client";
+import { BookPageProps } from "pages/[user]/book";
+import { HashLinkPageProps } from "pages/d/[link]/book";
+import { AvailabilityTeamPageProps } from "pages/team/[slug]/[type]";
+import { TeamBookingPageProps } from "pages/team/[slug]/book";
+import { FC, ReactNode } from "react";
+
+import { classNames } from "@calcom/lib";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { Icon } from "@calcom/ui/Icon";
+
+import { UserAvatars } from "@components/booking/UserAvatars";
+import EventTypeDescriptionSafeHTML from "@components/eventtype/EventTypeDescriptionSafeHTML";
+
+import type { AvailabilityPageProps } from "../../pages/[user]/[type]";
+import type { DynamicAvailabilityPageProps } from "../../pages/d/[link]/[slug]";
```

Comment:
same here

Reply:
Here's fine tho since it's already relative and we're only importing types. No runtime code here.